apps/a11y-front/src/app/pages/news-page/news-page.component.ts

```diff
@@ -1,24 +1,30 @@
-import { Component, inject } from '@angular/core';
-import { CommonModule } from '@angular/common';
+import { Component, inject, LOCALE_ID } from '@angular/core';
+import { CommonModule, registerLocaleData } from '@angular/common';
 import { HeaderComponent } from '../../components/header/header.component';
 import { FooterComponent } from '../../components/footer/footer.component';
 import { NewsService } from './news.service';
 import { TranslateModule, TranslateService } from '@ngx-translate/core';
+import { DatePipe } from '@angular/common';
+import localeFr from '@angular/common/locales/fr';
+
+registerLocaleData(localeFr);
 
 @Component({
   selector: 'a11y-news-page',
   standalone: true,
   imports: [CommonModule, HeaderComponent, FooterComponent, TranslateModule],
   templateUrl: './news-page.component.html',
   styleUrls: ['./news-page.component.scss'],
+  providers: [DatePipe, { provide: LOCALE_ID, useValue: 'fr' }],
 })
 export class NewsPageComponent {
-  constructor() {
+  constructor(private datePipe: DatePipe) {
     this.setNotifications();
   }
 
   private readonly newsService = inject(NewsService);
   private translate = inject(TranslateService);
+  protected actualLang = this.translate.currentLang === 'en' ? 'en' : 'fr';
```

Comment:
Avec cette condition, la langue par défaut est le français.
```suggestion
  protected actualLang = this.translate.currentLang === 'fr' ? 'fr' : 'en';
```